tests/javascripts/authenticateSecurityKey.test.js

```diff
@@ -131,11 +128,11 @@ describe('Authenticate with security key', () => {
       .mockImplementationOnce((url, options = {}) => {
         // subsequent POST of credential data to server
         expect(url.toString()).toEqual(
-          'https://www.notifications.service.gov.uk/webauthn/authenticate?next=%2Ffoo%3Fbar%3Dbaz'
+          'https://www.notifications.service.gov.uk/webauthn/authenticate'
```

Comment:
Without this, is the test still testing the post-authentication redirect?

Reply:
I think I have a new way to test this.